virtManager/delete.py

```diff
@@ -31,6 +31,12 @@
 
 
 class vmmDeleteDialog(vmmGObjectUI):
+    disk = None
+
+    @classmethod
+    def set_disk(cls, disk):
+        cls.disk = disk
+
```

Comment:
You don't want to use a @classmethod here. That means that you are setting vmmDeleteDialog.disk, but you want to be setting self.disk. So this should be a regular method:
```python
def set_disk(self, disk):
    self.disk = disk
````